Docs/source/standard/constants.rst

```diff
@@ -0,0 +1,13 @@
+Constants
+=========
+
+For convenience, the PICMI interface defines the following constants,
+which can be used directly inside any PICMI script.
+
+- ``picmi.constants.c``: The speed of light in vacuum.
+- ``picmi.constants.ep0``: The vacuum permittivity :math:`\epsilon_0`
+- ``picmi.constants.mu0``: The vacuum permeability :math:`\mu_0`
+- ``picmi.constants.q_e``: The elementary charge (absolute value of the charge of an electron).
+- ``picmi.constants.m_e``: The electron mass
+- ``picmi.constants.m_p``: The proton mass
+- ``picmi.constants.pi``: The number :math:`\pi`.
```

Comment:
Should we state that these are by default in SI and provide some default values?

Reply:
Hm, not sure. Right now, the PICMI standard does not say anything (I think) about units. Maybe we should do a separate PR, where we clearly state what the unit system/policy should be (everywhere not only for the constants)

Reply:
I agree that is should state that these should be in SI.

I don't think that this should include pi, since that comes with Python already, in math.pi.